### PR TITLE
chore: update ocabundleslist.json path

### PR DESCRIPTION
**Pull request by amanji**
Adds VSCode debug launch configuration.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (swcurran) [inline .vscode/launch.json:1]: Did you mean to include this file, or should it be gitignore’d?

File: app/lib/data.ts [hunk 2]
```diff
@@ -1,21 +1,16 @@
...
 
-export const BUNDLE_LIST_URL = "https://raw.githubusercontent.com/bcgov/aries-oca-bundles/main";
-export const BUNDLE_LIST_PATH = "/ocabundleslist.json";
+export const BUNDLE_LIST_URL = "https://bcgov.github.io/aries-oca-bundles";
+export const BUNDLE_LIST_FILE = "ocabundleslist.json";
 
 export async function fetchOverlayBundleList() {
   try {
-    const response = await fetch(BUNDLE_LIST_URL + BUNDLE_LIST_PATH);
+    const response = await fetch(BUNDLE_LIST_URL + "/" + BUNDLE_LIST_FILE);
```
Comment (swcurran) [inline app/lib/data.ts:8]: Why the switch?  Just curious...

Comment (amanji) [reply app/lib/data.ts:8]: More consistent with how I'm constructing paths in other parts of the code

File: .vscode/launch.json (hunk 1 above)
Comment (amanji) [reply .vscode/launch.json:1]: Yeah it adds to a better developer experience.

Comment (amanji) [reply .vscode/launch.json:1]: It's a debug configuration for VSCode.

Comment (swcurran) [reply .vscode/launch.json:1]: And you even mentioned it in the PR description :-)

**Review by swcurran — APPROVED** [review]

**Pull request merged**